modules/account-lib/src/coin/hbar/tokenTransferBuilder.ts

```diff
@@ -0,0 +1,104 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import * as Long from 'long';
+import { TokenId } from '@hashgraph/sdk';
+import { proto } from '../../../resources/hbar/protobuf/hedera';
+import { BuildTransactionError, InvalidParameterValueError } from '@bitgo/sdk-core';
+import { TransferBuilder } from './transferBuilder';
+import { Transaction } from './transaction';
+import { getHbarTokenFromTokenName, isValidToken, stringifyAccountId, stringifyTokenId } from './utils';
+
+export class TokenTransferBuilder extends TransferBuilder {
```

Comment:
What's the reason for extending `TransferBuilder` here instead of `TransactionBuilder`? All of the builders thus far have extended Transaction.